src/styles/_stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+  width: 96px;
+  justify-content: space-between;
+
+  &__star {
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+  }
+  &__star:nth-child(-n+4) {
```

Comment:
apply styles from previous (`Stars`) task. Should be styles for all moidifiers

Reply:
Still not fixed, you should add modifiers not only for 4 stars active but for any count like in Stars task